src/Uno.UWP/UI/Colors.cs

```diff
@@ -35,21 +33,153 @@ public static Color Parse(string colorCode)
+						nameof(LightPink) => 0xFFFFB6C1,
+						nameof(LightSalmon) => 0xFFFFA07A,
+						nameof(LightSeaGreen) => 0xFF20B2AA,
+						nameof(LightSkyBlue) => 0xFF87CEFA,
+						nameof(LightSlateGray) => 0xFF778899,
+						nameof(LightSteelBlue) => 0xFFB0C4DE,
+						nameof(LightYellow) => 0xFFFFFFE0,
+						nameof(Lime) => 0xFF00FF00,
+						nameof(LimeGreen) => 0xFF32CD32,
+						nameof(Linen) => 0xFFFAF0E6,
+						nameof(Magenta) => 0xFFFF00FF,
+						nameof(Maroon) => 0xFF800000,
+						nameof(MediumAquamarine) => 0xFF66CDAA,
+						nameof(MediumBlue) => 0xFF0000CD,
+						nameof(MediumOrchid) => 0xFFBA55D3,
+						nameof(MediumPurple) => 0xFF9370DB,
+						nameof(MediumSeaGreen) => 0xFF3CB371,
+						nameof(MediumSlateBlue) => 0xFF7B68EE,
+						nameof(MediumSpringGreen) => 0xFF00FA9A,
+						nameof(MediumTurquoise) => 0xFF48D1CC,
+						nameof(MediumVioletRed) => 0xFFC71585,
+						nameof(MidnightBlue) => 0xFF191970,
+						nameof(MintCream) => 0xFFF5FFFA,
+						nameof(MistyRose) => 0xFFFFE4E1,
+						nameof(Moccasin) => 0xFFFFE4B5,
+						nameof(NavajoWhite) => 0xFFFFDEAD,
+						nameof(Navy) => 0xFF000080,
+						nameof(OldLace) => 0xFFFDF5E6,
+						nameof(Olive) => 0xFF808000,
+						nameof(OliveDrab) => 0xFF6B8E23,
+						nameof(Orange) => 0xFFFFA500,
+						nameof(OrangeRed) => 0xFFFF4500,
+						nameof(Orchid) => 0xFFDA70D6,
+						nameof(PaleGoldenrod) => 0xFFEEE8AA,
+						nameof(PaleGreen) => 0xFF98FB98,
+						nameof(PaleTurquoise) => 0xFFAFEEEE,
+						nameof(PaleVioletRed) => 0xFFDB7093,
+						nameof(PapayaWhip) => 0xFFFFEFD5,
+						nameof(PeachPuff) => 0xFFFFDAB9,
+						nameof(Peru) => 0xFFCD853F,
+						nameof(Pink) => 0xFFFFC0CB,
+						nameof(Plum) => 0xFFDDA0DD,
+						nameof(PowderBlue) => 0xFFB0E0E6,
+						nameof(Purple) => 0xFF800080,
+						nameof(Red) => 0xFFFF0000,
+						nameof(RosyBrown) => 0xFFBC8F8F,
+						nameof(RoyalBlue) => 0xFF4169E1,
+						nameof(SaddleBrown) => 0xFF8B4513,
+						nameof(Salmon) => 0xFFFA8072,
+						nameof(SandyBrown) => 0xFFF4A460,
+						nameof(SeaGreen) => 0xFF2E8B57,
+						nameof(SeaShell) => 0xFFFFF5EE,
+						nameof(Sienna) => 0xFFA0522D,
+						nameof(Silver) => 0xFFC0C0C0,
+						nameof(SkyBlue) => 0xFF87CEEB,
+						nameof(SlateBlue) => 0xFF6A5ACD,
+						nameof(SlateGray) => 0xFF708090,
+						nameof(Snow) => 0xFFFFFAFA,
+						nameof(SpringGreen) => 0xFF00FF7F,
+						nameof(SteelBlue) => 0xFF4682B4,
+						nameof(Tan) => 0xFFD2B48C,
+						nameof(Teal) => 0xFF008080,
+						nameof(Thistle) => 0xFFD8BFD8,
+						nameof(Tomato) => 0xFFFF6347,
+						nameof(Turquoise) => 0xFF40E0D0,
+						nameof(Violet) => 0xFFEE82EE,
+						nameof(Wheat) => 0xFFF5DEB3,
+						nameof(White) => 0xFFFFFFFF,
+						nameof(WhiteSmoke) => 0xFFF5F5F5,
+						nameof(Yellow) => 0xFFFFFF00,
+						nameof(YellowGreen) => 0xFF9ACD32,
+						_ => throw new InvalidOperationException($"The color {colorCode} is unknown")
+					};
 
-					return color;
+					return new Color(color);
```

Comment:
couldnt you just return the static properties, like?:
```cs
return colorCode switch
{
	"transparent" => Colors.Transparent,
};
```
not sure if this is more or less performant~?

Reply:
Performance wise it should be identical (inlined).

However calling the `.ctor` makes it possible for the linker to remove the properties (when they are not used elsewhere in the app). Using them her would ensure they are not removable in any case.